pyproject.toml

```diff
@@ -16,12 +16,12 @@ data-simulator = 'datasimulator.main:main'
 [tool.poetry.dependencies]
 python = ">=3.9,<4"
 cdislogging = "*"
-gen3datamodel = {git = "https://github.com/uc-cdis/gdcdatamodel", rev = "3.1.3"}
+gen3datamodel = {git = "https://github.com/uc-cdis/gen3datamodel", rev = "3.2.0"}
 gen3dictionary = "*"
 pyyaml = "<7"
 rstr = ">=3,<4"
 requests = "*"
-dictionaryutils = "*"
+dictionaryutils = {git = "https://github.com/uc-cdis/dictionaryutils", branch = "chore/update-jsonschema"}
```

Comment:
Do we have a new release of dictionaryutils that uses a release from this merged branch? https://github.com/uc-cdis/dictionaryutils/commit/cbac4bf51febc64eab4875f84ca6cb15c8b4718e

Reply:
Yes, there is a new release of dictionaryutils. I was getting poetry errors when trying to update to the new release due to a cyclic dependency. I think I have to do another round of PRs where the dev-dependency of data-simulator is removed from dictionaryutils.